python/ray/tune/tests/test_task_events.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import sys
+
+import ray
+from ray._private.test_utils import (
+    run_string_as_driver,
+    wait_for_condition,
+)
+from ray.util.state import list_tasks
+
+
+@pytest.fixture
+def ray_init_4_cpus():
+    yield ray.init(num_cpus=4)
+    ray.shutdown()
+
+
+def test_no_missing_parent_task_ids(ray_init_4_cpus):
```

Comment:
I might be missing some background context, but why is this being moved to a tune-specific test? The original test seems to be general to core?

Reply:
the test is running tune code; we want to have no dependencies on library tests within core

the alternative is to delete this test, which I would also be ok with

Reply:
Oh, did you mean to delete `test_parent_task_id_tune_e2e` instead of `test_parent_task_id_threaded_task` in the original core test?

Reply:
I am ok to remove the tune one

Reply:
oh wow...... thanks for the catch!

Reply:
I replaced the `test_parent_task_id_threaded_task` test

Do you prefer to delete this test rather than moving it to tune dir?

Reply:
Haha thanks. Yeah I'd prefer to delete - I don't think there's anything specific to Tune that was captured in this test, and if Tune was doing something special for this test case I'd prefer to have a more isolated test for the interaction point rather than this e2e one.

Reply:
completely agree

Reply:
removing and updating description